Jenkinsfile

```diff
@@ -107,4 +108,10 @@ pipeline {
             }
         }
     }
+
+    post {
+        always {
+            notify("${NOTIFICATION_CHANNEL}", "${currentBuild.result} ${env.JOB_NAME} ${env.BRANCH_NAME}")
```

Comment:
I think this should be a richer message, with links to as many useful resources as possible.  Maybe something like:

>@hkeeler deployed PR #5647 via Jenkins job [cfgov-refresh » PR-5647 » #4]()
>
>View site at: https://cfgov-pr-5647...
>View stack at: https://docker-cluster/stacks/cfgov-pr-5647/...

I also think you might want to spit up the `always` into `success` and `unsuccessful` conditions, as I think the messages will likely be pretty different.

Also also, we may want to incorporate this into the shared library itself.  I'm guessing the patterns will be pretty similar across apps, and it'd cut down on a fair bit of boilerplate if we did that.

Reply:
@hkeeler how would you get the url to the PR inside the Pipeline?